code/workspace/package-and-deploy/scripts/deploy-cfn-template.sh

```diff
@@ -2,28 +2,27 @@
 
 # From code/package-and-deploy directory run ./scripts/deploy-cfn-template.sh
 
-BUCKET_NAME="cfn-workshop-s3-s3bucket-1wo3wjmf93viq" # use your own uniquely named S3 bucket
-PREFIX_NAME="lambda-function"
-STACK_NAME="cfn-workshop-lambda"
+BUCKET_NAME="rezabekf-development-bucket" # cfn-workshop-s3bucket-unique-name # use your own uniquely named S3 bucket
```

Comment:
same thing here with requiring this as an arg

Reply:
Sorted